website/src/app/kb/quickstart/readme.mdx

```diff
@@ -0,0 +1,88 @@
+import Image from "next/image";
+
+# Quickstart Guide
+
+Welcome to the quickstart guide for Firezone
+
+<Image
+  src="/images/firezone-v1-architecture-simple.png"
+  alt="Firezone simplified architecture"
+  width={900}
+  height={50}
+  className="mx-auto shadow rounded"
+/>
+
+
+## Prerequisites
+
+* Firezone account (Don't have an account? Get one here https://app.firezone.dev/sign_up)
+* Resource you want to give users secure access to (e.g. prod server, database, SaaS application, subnet, etc...)
+* Server or VM (with the ability to deploy a docker container or Linux application) that can connect to the resource, and the internet
```

Comment:
would "to both the resource, and the internet" be more clear here?

Reply:
Sounds good to me 👍 
I'll get it updated.